src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,64 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+    private static final double MAG_TICKS = 4096;
+    private static final double FALCON_TICKS = 2048;
+
+    public static double magTicksToDegrees(double magTicks) {
+        return magTicksToRevolutions(magTicks) * 360;
+    }
+
+    public static double degreesToMagTicks(double degrees) {
+        return degreesToRevolutions(degrees) * MAG_TICKS;
+    }
+
+    public static double degreesToRevolutions(double degrees) {
+        return degrees / 360;
+    }
+
+    public static double revolutionsToDegrees(double revolutions) {
+        return revolutions * 360;
+    }
+
+    public static double revolutionsToMagTicks(double revolutions) {
+        return revolutions * MAG_TICKS;
+    }
+
+    public static double magTicksToRevolutions(double magTicks) {
+        return magTicks / MAG_TICKS;
+    }
+
+    public static double hundredMsToSeconds(double hundredMs ) {
+        return hundredMs * 10;
+    }
+
+    public static double falconTicksToRevolutions(double ticks) {
```

Comment:
Something about this function is wrong. Check again what's the point of this and if the name/calculation is correct